### PR TITLE
Hero page pr6

### PR DESCRIPTION
**Pull request by RainingComputers**

### CODE REVIEW

File: meiki_ui/src/components/Button.svelte [hunk 1]
```diff
@@ -1,15 +1,18 @@
 <script context="module" lang="ts">
     export enum ButtonType {
+        HERO,
         PRIMARY,
```
Comment (alexsanjoseph) [inline meiki_ui/src/components/Button.svelte:4]: wonder if we should call something other than Primary/Secondary (especially secondary)

**Comment by alexsanjoseph** [comment]
LGTM

**Comment by alexsanjoseph** [comment]
If you're logged in, you can never see the Hero page, huh?

File: meiki_ui/src/components/Button.svelte (hunk 1 above)
Comment (RainingComputers) [reply meiki_ui/src/components/Button.svelte:4]: Hmm, yep we just kind of assumed this, we don't know what it really means. Will look into UI jargons for this.

**Comment by RainingComputers** [comment]
Yeah, I think that is how google drive also behaves.
If the user really loves our hero page he can always use the incognito tab :P

**Pull request merged**